js/module.js

```diff
@@ -0,0 +1,156 @@
+  {
+    imgSrc: './assets/speakers/omos.jpg',
+    name: 'Stanley Osagie',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/Tosin.jpg',
+    name: 'Tosin Adewale',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/mama.jpg',
+    name: 'Samuel Juliana',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/omos.jpg',
+    name: 'Omos Ehigbe',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/Tosin.jpg',
+    name: 'Iya Alata',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+  {
+    imgSrc: './assets/speakers/mama.jpg',
+    name: 'Mama Omos',
+    qualities: 'Lorem ipsum dolor sit, amet consectetur adipisicing.',
+    speech: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptate voluptates ipsa libero enim deserunt quia sunt quam laudantium nisi veniam, asperiores neque sed quaerat.',
+  },
+];
+
+const sections = {
+  program: document.querySelector('#main-program'),
+  join: document.querySelector('join'),
+  sponsor: document.querySelector('#sponsors'),
+  news: document.querySelector('#news'),
+};
+
+const checkbox = document.querySelector('#menu-checkbox');
+const body = document.querySelector('body');
+
+function createSpeakerElement(speaker) {
+  const article = document.createElement('article');
+  article.classList = 'speaker';
+
+  const img = document.createElement('img');
+  img.className = 'spk-img';
+  img.src = speaker.imgSrc;
+  img.alt = speaker.name;
+
+  const section = document.createElement('section');
+  section.className = 'spk-bio';
+
+  const h2 = document.createElement('h2');
+  h2.className = 'spk-name';
+  h2.textContent = speaker.name;
+
+  const pQualities = document.createElement('p');
+  pQualities.className = 'spk-qualities';
+  pQualities.textContent = speaker.qualities;
```

Comment:
- [ ] 👏👏 Great job on choosing the perfect color! However, as per the design, the text should be displayed in italics and bold. To match the design requirements, please update the text to be displayed in italic  and bold format.

By making this adjustment, your text will look more in line with the design and enhance the overall visual appeal of your project. Keep up the good work and happy coding! 🌟💻🚀